tracing-subscriber/src/reload.rs

```diff
@@ -11,9 +11,29 @@
 //!
 //! # Examples
 //!
-//! Reloading a [`Filtered`](crate::filter::Filtered) layer to change the filter at runtime.
-//!
+//! Reloading a global filtering layer:
```

Comment:
nit: maybe worth adding a link to the top-level filtering documentation here?
```suggestion
//! Reloading a [global filtering](crate::layer#global-filtering) layer:
```